src/sync/incomingSyncUtils.js

```diff
@@ -249,14 +251,18 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
       cannotBeBlank: [],
       canBeBlank: ['units', 'comment', 'order_number'],
     },
+    Report: {
+      cannotBeBlank: ['ID', 'title', 'type', '_data'],
```

Comment:
this will be the field coming FROM the server which would be `data`. We are using `_data` on mobile just to be difficult 🤣 but the field in the mSupply database will be `data`